plugins/balancedLevelProvider.go

```diff
@@ -59,6 +62,7 @@ func makeBalancedLevelProvider(
 	virtualBalanceQuote float64,
 	orderConstraints *model.OrderConstraints,
 ) api.LevelProvider {
+	l := logger.MakeBasicLogger()
```

Comment:
this should reuse the existing logger that is passed in. we should have only one call to MakeBasicLogger throughout Kelp

Reply:
done